src/components/Article.jsx

```diff
@@ -1,29 +1,57 @@
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
-import { getArticleById } from "../api";
+import { getArticleById, increaseVote } from "../api";
+
+import Comments from "./Comments";
 
 export default function Article() {
   const [article, setArticle] = useState({});
+  const [votes, setVotes] = useState(0);
   const [isLoading, setIsLoading] = useState(true);
   const { article_id } = useParams();
 
   useEffect(() => {
     getArticleById(article_id).then((res) => {
       setArticle(res);
       setIsLoading(false);
+      setVotes(res.votes);
     });
   }, [article_id]);
 
   if (isLoading) {
     return <p>Loading...</p>;
   }
+  const handleVote = (event) => {
+    event.preventDefault();
+    setVotes((currentVotes) => {
+      return currentVotes + 1;
+    });
+    increaseVote(article_id).catch((err) => {
+      setVotes((currentVotes) => {
+        return currentVotes - 1;
+      });
+      console.log(err);
```

Comment:
Rather than console logging the err, it would be good to notify the user if patch request fails (upvoting an article). Maybe a message saying something like 'upvoting was not possible at this time"